src/dispatch.rs

```diff
@@ -388,12 +388,25 @@ impl<'t> DispatcherBuilder<'t> {
     /// # Panics
     ///
     /// * if the specified dependency does not exist
-    pub fn add<T>(mut self, system: T, name: &str, dep: &[&str]) -> Self
+    pub fn add<T>(self, system: T, name: &str, dep: &[&str]) -> Self
```

Comment:
It's documented for `Resources`, `ResourceId` and `Resources::add`, so it should be quite obvious. If you still feel we should add it, I'd be happy to accept a PR.